src/transformers/integrations/awq.py

```diff
@@ -266,8 +270,9 @@ def fuse_awq_modules(model, quantization_config):
         # Replace layer norms
         _fuse_awq_layernorm(modules_to_fuse["layernorm"], module, FasterTransformerRMSNorm)
 
-        # Replace MLP layers
-        _fuse_awq_mlp(model, name, modules_to_fuse["mlp"], module, QuantFusedMLP)
+        # Replace MLP layers if awq version is not ipex.
+        if quantization_config.version != "ipex":
+            _fuse_awq_mlp(model, name, modules_to_fuse["mlp"], module, QuantFusedMLP)
```

Comment:
Need to add a logger.info to tell  the user that we didn't fuse the mlp with ipex. Also, for curiosity, why this doesn't work with ipex while for other layers with attention layers, it works

Reply:
That's because the fused mlp needs awq_ext which only supported in CUDA, see [here](https://github.com/casper-hansen/AutoAWQ/blob/main/awq/modules/fused/mlp.py#L37-L42). We'd like to add CPU support in the future, but for now, let's skip this feature.